include/lsst/meas/base/FlagHandler.h

```diff
@@ -39,6 +39,15 @@ namespace lsst { namespace meas { namespace base {
     C-strings so we can create these arrays using initializer lists even in C++98.
  */
 struct FlagDefinition {
+
+    FlagDefinition() {
+    }
+
+    FlagDefinition(char const * _name, char const * _doc) {
+        name = _name;
+        doc = _doc;
+    }
+
```

Comment:
I am slightly worried what is going to happen when you have python strings mapped to c style pointers. If the python strings somehow get cleaned up. What will happen to the pointers. Is it practical to use std::strings, or am I being overly worried?